document/src/vespa/document/bucket/bucketselector.cpp

```diff
@@ -81,51 +81,39 @@ using namespace document::select;
                      const select::Operator& op)
         {
             if (node.getType() == IdValueNode::ALL) {
-                const StringValueNode* val(
-                        dynamic_cast<const StringValueNode*>(&valnode));
+                auto val = dynamic_cast<const StringValueNode*>(&valnode);
```

Comment:
I personally prefer to explicitly indicate pointer/ref-ness with `auto*/auto&` to ensure no implicit value capture (not a problem with pointers, though)

Reply:
I do too unless they are primitives.